docs/DATABASE_FORMAT.md

```diff
@@ -0,0 +1,114 @@
+  
+  
+![Redundant Edges](./img/redundant_mesh.png)  
+  
+  
+## NAMING CONVENTIONS  
+Area names should be memorable and descriptive. If your game does not provide an official name for a room, you will need to name it yourself. Avoid using in-jokes that newer players may be unfamiliar with, and remember that while a funny room name now and then can be good, it should not come at the expense of utility. Needless to say, profane or offensive names will be rejected outright. If a game's community has already adopted community names for rooms, it is best to use those as well, as many resources will have already been created using them, and players familiar to the game will easily adapt to your database. It can occasionally be worthwhile to alter or ignore an established community name, however, if it comes into conflict with the aforementioned principles.  
+  
+Node names do not need to be memorable, and should be purely descriptive. Because an area contains only a few nodes, anyone reviewing the database will almost certainly know what each node refers to. In addition, there are a few naming conventions that exist for specific node types:  
+  
+Pickup nodes should follow the format  
+Pickup (Pickup Type)  
+  
+If two pickups in a room would share the same name, it is required that you add some descriptive text to their names to distinguish them. For example,  
+Pickup (Lower Ledge Missile Expansion)  
+Pickup (Morph Tunnel Missile Expansion)  
+  
+Event nodes should follow the format  
+Event - Event Name  
+  
+And doors should adhere to  
+Door to Room Name  
+  
+  
+## COMMENTS  
+Any block requirement can be supplemented with a comment. Comments can serve multiple purposes. Comments may be used as a way to justify design decisions to other Database creators, as a means to mark something for future review, or as a description of the block's logic, so that players and creators can understand how some sequence of actions can be performed.  
+  
+Comments should be at least somewhat professional in tone. It is not necessary that comments form full sentences or use formal language, but they should communicate their message and purpose clearly. Avoid leaving comments that you wouldn't want another player to see.  
+  
+  
+## TRICK VIDEO DOCUMENTATION  
+Comments, like most text in Randovania, will automatically hyperlink any links that are included. Although many tricks are obvious in their implementation, for tricks that are either obscure, difficult to perform, or not obvious to the novice, it is advised that you document such a trick. The best way to document a trick is to provide video footage of it being performed. Trick performances should ideally meet as many of the following criteria as possible:  
```

Comment:
Actually links have always to be manually made clickable. It's something I explicitly added for comments and maybe ticks?

Reply:
It's included for tricks and comments, which are the only places I've ever added a link, so I went and got myself a little confused.